ntoskrnl/ex/sysinfo.c

```diff
@@ -664,7 +664,11 @@ QSI_DEF(SystemProcessorInformation)
 #else
     Spi->MaximumProcessors = 0;
 #endif
+#if (NTDDI_VERSION >= NTDDI_WIN10) || ((NTDDI_VERSION >= NTDDI_WIN8) && defined(_WIN64))
```

Comment:
NTDDI_VERSION >= NTDDI_WIN8 condition already includes NTDDI_WIN10

Reply:
Yes, but not `_WIN64`

Reply:
Ah yes, the braces.